src/guides/v2.3/howdoi/admin/customize_admin.md

```diff
@@ -0,0 +1,233 @@
+---
+group: how-do-i
+subgroup: admin-customize-page
+title: Customize Magento Admin Design
+menu_order: 3
+---
+
+Learn how to customize (add, delete, change) the configuration options available in the Magento Admin. These options define the various aspects of storefront design.
+
+The location of your customizations appears here:
+
+-  Magento Commerce and Open Source 2.1.0 and later: Options set under **CONTENT > Design > Configuration**
+
+## Environment and technologies
+
+-  Magento Commerce
+-  Magento Commerce (Cloud)
+-  Magento Open Source
+
+## Prerequisites
+
+-  Magento installed
+-  Credentials or access to Magento Admin
+
+## Steps
+
+ {:.bs-callout-info}
+The following walk-through uses version 2.1.0 and later.
+
+In Magento out of the box, when you navigate to **CONTENT > Design > Configuration** in Admin, the first page that opens displays a grid with the available configuration scopes and assigned themes. It looks like following:
+
+![Design Configuration]({{ site.baseurl }}/common/images/design_conf1.png)
+
+When you click **Edit** in any of the scope records, the page with available design options is displayed. For example, the default set of design options for the store view level is the following:
+
+![Default Store View]({{ site.baseurl }}/common/images/design_conf2.png)
+
+Both the grid and the configuration form are implemented using UI components.
+
+To change the grid view, you need to [customize the grid](#customize-the-grid) configuration by adding a custom `design_config_listing.xml` to your module.
+
+To change the available design settings you need to [customize the grid](#customize-the-grid) configuration by adding your custom `design_config_form.xml` in your module. If you add a new field, you must also declare it in `di.xml` on how it is processed and saved.
+
+## Customize the grid
+The grid containing the configuration scopes is implemented using the [grid UI component]({{page.baseurl}}/ui_comp_guide/components/ui-listing-grid.html).
+
+To customize the grid view, take the following steps:
+
+1. In the `<your_module_dir>/view/adminhtml/ui_component` directory, add the empty `design_config_listing.xml`.
+
+1. In the `design_config_listing.xml` file, create an element to which to add your customizations. For example, if you want to rename the column displaying the selected theme, your grid configuration must contain the following:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This entire code block must be indented 3 spaces to align properly under the preceding list number.

Reply:
Resolved